api_helpers.py

```diff
@@ -475,7 +520,8 @@ def create_hs_res_from_czo_row(czo_res_dict, czo_hs_account_obj, index=-99, ):
         # rights, funding_agencies, extra_metadata
 
         _success_file = True
-        for f in get_files(czo_files, record_dict=record_dict):
+        other_urls = [] + map_uploads_list + kml_files_list
```

Comment:
more informative name than record_dict please that tells what inside it also don't use data type words in vars str, dict, list...